docs/how-tos/configure-teleop-workspace.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Configure a teleop workspace"
+linkTitle: "Configure a teleop workspace"
+weight: 10
+type: "docs"
+description: "Create and configure a teleop workspace with widgets."
+images: ["/how-tos/teleop/full-workspace.png"]
+icon: true
+tags: ["teleop", "configuration"]
+languages: []
+viamresources: ["sensor", "camera", "movement sensor"]
+platformarea: ["viz", "data"]
+level: "Intermediate"
+date: "2024-11-13"
+# updated: "2024-08-26"  # When the tutorial was last entirely checked
+cost: "0"
+---
+
+You can use teleop to create a custom workspace where you can visualize and aggregate data from a machine.
+You can currently visualize data from a camera, a sensor, or a movement sensor.
+
+{{% alert title="In this page" color="info" %}}
+
+- [Configure a workspace](#configure-a-workspace)
+
+{{% /alert %}}
+
+## Prerequisites
+
+{{% expand "A configured machine with teleoperable components" %}}
+
+Make sure your machine has at least one of the following:
+
+- A camera, movement sensor, sensor, base, arm, board, gantry, gripper, motor or servo
```

Comment:
updated this for actuation-- I figure not updating the screenshots to also include an actuation widget is fine for this demo as its pretty self explanatory but if anyone thinks differently I can update/retake the screenshots